eva/expression/expression_utils.py

```diff
@@ -22,31 +22,48 @@
 from eva.expression.tuple_value_expression import TupleValueExpression
 
 
-def expression_tree_to_conjunction_list(expression_tree):
+def to_conjunction_list(
+    expression_tree: AbstractExpression,
+) -> List[AbstractExpression]:
+    """Convert expression tree to list of conjuntives
+
+    Note: It does not normalize the expression tree before extracting the conjuntives.
+
+    Args:
+        expression_tree (AbstractExpression): expression tree to transform
+
+    Returns:
+        List[AbstractExpression]: list of conjunctives
+
+    Example:
+        to_conjunction_list(AND(AND(a,b), OR(c,d))): [a, b, OR(c,d)]
+        to_conjunction_list(OR(AND(a,b), c)): [OR(AND(a,b), c)]
+            returns the original expression, does not normalize
+    """
     expression_list = []
     if expression_tree.etype == ExpressionType.LOGICAL_AND:
-        expression_list.extend(
-            expression_tree_to_conjunction_list(expression_tree.children[0])
-        )
-        expression_list.extend(
-            expression_tree_to_conjunction_list(expression_tree.children[1])
-        )
+        expression_list.extend(to_conjunction_list(expression_tree.children[0]))
+        expression_list.extend(to_conjunction_list(expression_tree.children[1]))
     else:
         expression_list.append(expression_tree)
 
     return expression_list
 
 
-def conjuction_list_to_expression_tree(
+def and_(
```

Comment:
Any reason we change to this name? I think the original name is more intuitive.

Reply:
Sure, I'll revert back